CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ After this, `Decidim::Proposals::ProposalEndorsement` and the corresponding coun
 
 ### Fixed
 
+- **decidim-initiatives**: Fix initiative answer redirection in case of invalid form [\#6279](https://github.com/decidim/decidim/pull/6279)
```

Comment:
Hi @Quentinchampenois 
from recent release, Decidim has changed its CHANGELOG file policy as is stated here: #5908 
For this, could you remove this changelog file entry please ?
Thanks!

Reply:
Hi @Leusev,

I did not see this decision, I removed the entry in changelog file .

Thanks for the link and your suggestion !

Reply:
Don't worry, it is difficult to keep up with such changes! 
Thank you for your work! :smile: